src/status_im/ui/screens/discover/all_dapps/views.cljs

```diff
@@ -9,9 +9,12 @@
             [status-im.ui.screens.discover.components.views :as discover-components]
             [status-im.ui.screens.discover.styles :as styles]
             [status-im.i18n :as i18n]
-            [status-im.components.toolbar-new.view :as toolbar]))
+            [status-im.components.toolbar-new.view :as toolbar]
+            [taoensso.timbre :as log]))
 
 (defn render-dapp [{:keys [name photo-path dapp?] :as dapp}]
+  [react/touchable-highlight {:on-press #(do (re-frame/dispatch [:set :discover-current-dapp dapp])
```

Comment:
probably we could extract `do` form into separate function

Reply:
Right, it's used twice. Sure!